roadmaps/configuration.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: contribute
+title: Bazel Configurability Roadmap
+---
+<style>
+  .padbottom { padding-bottom: 10px; }
+  .etabox {
+    background: #EFEFEF;
+    color: #38761D;
+    font-size: 15px;
+    font-weight: bold;
+    display: inline;
+    padding: 6px;
+    margin-right: 10px;
+  }
+</style>
+
+# Bazel Configurability 2018 Roadmap
+*Last verified: Wednesday, June 6, 2018* ([update history]
+(https://github.com/bazelbuild/bazel-website/commits/master/roadmaps/configuration.md))
+
+This page describes committed plans to advance Bazel's configuration and
+multiplatform support in 2018.
+
+Dates are approximate based on our best understanding of problem complexity
+and developer availability. ETAs will change, but we'll keep them refreshed and
+current.
+
+Have a bug or feature request? File a [GitHub issue
+](https://github.com/bazelbuild/bazel/issues/new).
+
+Questions or suggestions? Email
+<a href="mailto:bazel-discuss@googlegroups.com">bazel-discuss@googlegroups.com</a>.
+
+
+* [Goal](#goal)
+* [Roadmap](#roadmap)
+  * [Platforms](#platforms)
+  * [User-Defined Configuration](#user-defined-configuration)
+  * [Correctness and Speed](#correctness-and-speed)
+
+
+## Goal 
+
+Configurability's goal is to make Bazel a graceful multiplatform build
+tool. It also focuses on how users decide how their projects are built.
+
+
+This translates into the following high-level goals:
+
+1. **`$ bazel {build|test} :all` just works**
+    1. All targets "know" how to build for the right platforms with the right
+       toolchains and desired options
+    1. <div class="padbottom">Users only have to fiddle with options they care
+       about</div>
+
+1. **"Platforms" is a first-class concept**
+    1. "Platforms" and "toolchains" are well-defined, map well to reality, and are
+        easy to create  
+    1. <div class="padbottom">Builds, build rules, remote executors, etc.
+       naturally integrate platforms</div>
+
+1. **Users decide what to configure**
+
+    `$ bazel build //myapp:fancy_edition # automatically builds my app with "fancy" features`
```

Comment:
Remove the Bash comment from the command to keep the command clean. Write the comment as text above or below the command.